content/en/blog/_posts/2021-12-15-pod-security-admission-psa-beta.md

```diff
@@ -0,0 +1,620 @@
+```
+
+This is because the restricted profile explicitly requires that certain values are set to the most secure parameters.
+
+By requiring explicit values, manifests become more declarative and your entire security model can shift left. With the `restricted` level of enforcement, a company could audit their cluster's compliance based on permitted manifests.
+
+Let's fix each warning resulting in the following file:
+
+```shell
+cat <<EOF | kubectl -n kube-test-restricted apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-restricted
+spec:
+  containers:
+  - name: nginx
+    image: nginx:alpine
+    securityContext:
+      seccompProfile:
+        type: RuntimeDefault
+      runAsNonRoot: true
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+          - ALL
+        add:
+          - NET_BIND_SERVICE
+EOF
+```
+
+The output is similar to this:
+
+```shell
+pod/nginx-restricted created
+```
+
+At this point, if you wanted to dive deeper into permissions and what is or isn't enabled on a certain workload, exec into the control plane and play around `containerd`+`crictl inspect` with:
+
+```shell
+docker exec -it kind-control-plane bash
+
+# list running containers
+crictl ps
+
+# inspect each one by container ID
+crictl inspect <CONTAINER ID>
+```
+
+### Applying a cluster-wide policy
+
+In addition to applying labels to namespaces to configure policy you can also configure cluster-wide policies and exemptions using the AdmissionConfiguration resource. 
+
+Using this resource, policy definitions are applied cluster-wide by default and any policy that is applied via namespace labels will take precedence. 
+
+There is no runtime configurable API for this resource and a cluster administrator would need to specify a path to the file below via the `--admission-control-config-file` flag on the API server. 
+
+In the following resource we are enforcing the baseline policy and warning and auditing the baseline policy. We are also making the kube-system namespace exempt from this policy.
+
+It's not recommended to alter control plane / clusters after install, so let's build a new cluster with a default policy on all namespaces.
+
+First, delete the current cluster.
+
+```shell
+kind delete cluster
+```
+
+Create a Pod Security configuration. 
+
+<!-- TODO: link to the official docs after posted: https://github.com/kubernetes/website/blob/dev-1.23/content/en/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md?plain=1#L25-L55
+--->
+
+<!-- TODO: is this a BUG?:
+It errors with runtimeClassNames: [] and runs without it, tested on both alphav1/betav1 apis.
```

Comment:
Looks like you already corrected the fieldname to `runtimeClasses` (I think we had mis-documented it)